pandas/core/generic.py

```diff
@@ -6673,9 +6673,7 @@ def convert_dtypes(
             :class:`Series` (still experimental). Behaviour is as follows:
 
             * ``"numpy_nullable"``: returns nullable-dtype-backed :class:`DataFrame`
-              or :class:`Series` (default).
-            * ``"pyarrow"``: returns pyarrow-backed nullable :class:`ArrowDtype`
-              DataFrame or Series.
+            * ``"pyarrow"``: returns with pyarrow-backed nullable :class:`ArrowDtype`
 
```

Comment:
`or Series` needs to remain. Also, the 2nd line should be a close mirror of the first:

```
* ``"pyarrow"``: returns pyarrow-backed nullable :class:`ArrowDtype` :class:`DataFrame` or :class:`Series`.
```